client/app/dbaas/dbaas-metrics/token/metrics-token.html

```diff
@@ -4,46 +4,44 @@ <h3 class="oui-header_3" data-translate="metrics_token_title"></h3>
-            data-ng-disabled="MetricsTokenCtrl.actions.loading"
-            data-translate="metrics_token_add_token"
-            ui-sref="dbaas.metrics.detail.token.add({ serviceName: MetricsTokenCtrl.serviceName })"></a>
-        <div class="pull-right">
-            <input type="search"
-                class="oui-input"
-                id="token-search"
-                name="token-search"
-                data-ng-change="MetricsTokenCtrl.onSearchText()"
-                data-ng-model="MetricsTokenCtrl.searchText"
-                data-ng-model-options="{ debounce: 400 }"
-                data-translate-attr="{ placeholder: 'metrics_token_search' }"/>
-        </div>
-    </div>
-    <div class="metrics__row">
-        <oui-table id="token-table"
-                   rows="MetricsTokenCtrl.tokens"
-                   rows-loader="MetricsTokenCtrl.loading">
-            <column title="'metrics_table_name' | translate"
-                    property="description"
-                    sortable>
-                <span data-ng-bind="$row.description || '-'"></span>
-            </column>
-            <column title="'metrics_token_labels' | translate"
-                    property="labels"
-                    template="MetricsTokenCtrl.labelsTemplate()">
-            </column>
-            <column title="'metrics_table_token' | translate"
-                    property="access">
-                <cui-clipboard data-text="$row.access"></cui-clipboard>
-            </column>
-            <column title="'metrics_token_permission' | translate"
-                    property="permission">
+        <oui-datagrid rows="MetricsTokenCtrl.tokens">
+            <extra-top>
+                <button class="oui-button oui-button_secondary" type="button"
+                    data-ng-disabled="MetricsTokenCtrl.actions.loading"
+                    data-translate="metrics_token_add_token"
+                    ui-sref="dbaas.metrics.detail.token.add({ serviceName: MetricsTokenCtrl.serviceName })"></button>
+            </extra-top>
+            <oui-column title="'metrics_table_name' | translate" sortable="asc" searchable property="description" type="string">
+                    <span data-ng-bind="$row.description || '-'"></span>
+            </oui-column>
+            <oui-column title="'metrics_token_labels' | translate" type="string" property="labels">
+                <span class="token-labels" data-ng-repeat="label in $row.labels | orderBy : 'key'" data-ng-if="$index < 2">
+                    <span data-ng-bind="label.key"></span>:<span data-ng-bind="label.value"></span>
+                </span>
+                <oui-dropdown data-ng-if="$row.labels.length > 2">
+                    <div class="token-labels__button-container">
+                        <button type="button" class="token-labels token-labels__button" data-ng-bind="MetricsTokenCtrl.displayRemainingLabels($row.labels.length - 2)" oui-dropdown-trigger></button>
```

Comment:
`oui-dropdown-trigger` ?

Reply:
Yup, there is a dropdown if thecount of labels is more than 2

Reply:
Alright, I didn't know about `oui-dropdown-trigger` 😅